tests/test_offices.py

```diff
@@ -9,13 +9,18 @@ class TestOfficeActions():
     default_usa_state = "MA"
     default_office_code = default_usa_state + "-18"
     default_office_name = "Office "+default_office_code
+
     default_usa_state2 = "NH"
     default_office_code2 = default_usa_state2 + "-03"
     default_office_name2 = "Office "+default_office_code2
+
+    default_hoss_usa_state = "HOSS"
```

Comment:
I’m not a DB expert but just wanted to make sure we don’t want to make the usa_state for HOSS `null`? 
https://github.com/codefordc/us-congress-pizza-flag-tracker/pull/8/files/c5aeacc2b[…]2646a639abb4873b716b2..e93a80925feac95915e8a8634f5fc26b47ed70e5

Otherwise I think the PR looks good. Thanks again for the hard work!

I mean specifically we will soon need to filter out offices like HOSS and Mail Services from the list of usa_states and associated offices to return to the front end. 

Making HOSS’s USA_State HOSS means that we might lack a way to filter out federal agencies when we write that query to generate this JSON/dict: 
https://github.com/codefordc/us-congress-pizza-flag-tracker-frontend/blob/a2af9a893fcb2cc3269b0e347ccacac2fc3a243f/src/components/states.js 

`{"name:"HOSS", "districts": ["HOSS-1"] }` could easily sneak in